lizmap/modules/view/controllers/lizMap.classic.php

```diff
@@ -257,11 +257,20 @@ function f($x)
         if ($wmsInfo['WMSServiceTitle'] != '') {
             $title = $wmsInfo['WMSServiceTitle'];
         }
+        $openGraphTitle = $title;
 
         $title .= ' - '.$lrep->getLabel();
         $title .= ' - '.$lser->appName;
         $rep->title = $title;
 
+        // use addHeadContent to prevent url html escaping
+        $rep->addHeadContent('<meta property="og:image" content="'.jUrl::getFull('view~media:illustration', array('repository' => $this->param('repository'), 'project' => $this->param('project'))).'">');
+        $rep->addMeta(array('property' => 'og:title', 'content' => $openGraphTitle));
+        $rep->addMeta(array('property' => 'og:image:width', 'content' => '250'));
+        $rep->addMeta(array('property' => 'og:image:height', 'content' => '250'));
+        if (!empty($wmsInfo['WMSServiceAbstract'])) {
+            $rep->addMeta(array('property' => 'og:description', 'content' => $wmsInfo['WMSServiceAbstract']));
```

Comment:
The abstract can contain HTML element, is it automatically cleaned ?

Reply:
good point, the content is escaped , but not sure if cleaned, i check

Reply:
I checked the html tags are removed